supply-chain/audits.toml

```diff
@@ -0,0 +1,4 @@
+
+# cargo-vet audits file
```

Comment:
how does this work for configuration ?

Reply:
cargo-vet needs it to perform the audit (store acquisition).